pages/addboard.tsx

```diff
@@ -0,0 +1,90 @@
+import FileInput from "@/components/FileInput/FileInput";
+import styles from "./AddBoard.module.css";
+import { useReducer, useState } from "react";
+import { useRouter } from "next/router";
+
+interface ArticleInfo {
+  articleTitle: string;
+  articleContent: string;
+  articleImage: File | null;
+}
+
+const AddBoard = () => {
+  const [articleInfo, setArticleInfo] = useState<ArticleInfo>({
+    articleTitle: "",
+    articleContent: "",
+    articleImage: null,
+  });
+  const router = useRouter();
+
+  const handleInputChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const { name, value } = e.target;
+    handleChange(name, value);
+  };
+
+  const handleChange = (name: string, value: string | File | null) => {
+    setArticleInfo((prevInfo) => ({
+      ...prevInfo,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = () => {
+    console.log("submit 확인");
+    //폼 제출 로직...
+    //페이지 이동....
+  };
```

Comment:
```suggestion
  const handleSubmit = () => {
    console.log("submit 확인");
/**
 * @TODO 이런거 고쳐야함
 */
  };
```
위처럼 주석을 쓰시면 조금 더 직관적으로 주석을 표현할 수 있어요 한번 붙여넣기 해보세요!